lib/spack/spack/binary_distribution.py

```diff
@@ -98,41 +96,36 @@ def read_buildinfo_file(prefix):
     return buildinfo
 
 
-def _find_relocations(prefix):
+def write_buildinfo_file(prefix, workdir, rel=False):
+    """
+    Create a cache file containing information
```

Comment:
This will probably fail the documentation tests. Docstrings aren't allowed to start with a newline character.

Reply:
Is this recent? The doc tests did pass in this case (although the Mac tests keep failing on startup).